tests/README.md

```diff
@@ -0,0 +1,8 @@
+﻿# Tests
+
+Contains Pester tests for PowerShell code in the module, and a Module.Tests.ps1 file. 
+The module tests file runs ScriptAnalyzer tests against the module.
```

Comment:
This needs to be expanded slightly because currently, those common tests don't exist anywhere public -- we could contribute them to [PoshCode\ModuleBuilder](https://github.com/poshcode/modulebuilder) or something -- but for now we should at least explain that:

We reference the ScriptAnalyzer tests and common module functionality tests from our `build` module because that way we're always automatically using the `latest` version of the shared test code -- rather than getting a point-in-time copy when we clone the module template.